packages/next/types/index.d.ts

```diff
@@ -48,6 +48,9 @@ export type NextPage<P = {}, IP = P> = {
  */
 export type PageConfig = {
   amp?: boolean | 'hybrid'
+  api?: {
+    bodyParser?: boolean
```

Comment:
The problem with having a static export is that it always has to be configure by the user. Third party libraries can't configure the handlers they produce to opt out of this body-parser. e.g. if I create a proxy library, I'd have to ask my users to always include this static export:
```js
import someProxy from 'third-party';
export default someProxy();
// I'd have to ask users of my library to include the following or it won't work:
export const {
  api: {
    bodyParser: false
  }
}
```